addons/website/views/snippets/s_quotes_carousel_minimal.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_quotes_carousel_minimal" name="Quotes Minimal">
+    <section class="s_quotes_carousel_minimal_wrapper">
```

Comment:
```suggestion
<template id="s_quotes_carousel_minimal" name="Quotes Minimal">
    <section class="s_quotes_carousel_wrapper" data-vxml="001" data-vcss="002">
```
No need to introduce a new snippet class for this snippet, it's actually an `s_quotes_carousel_wrapper` with some options changed 😉. This will avoid copying and pasting CSS, adding selectors in options, etc..
I'll continue to review the PR after the change

Reply:
Thanks @bvr-odoo, I adapted the PR, could you have a look at it please?